cms/static/coffee/spec/models/textbook_spec.js

```diff
@@ -0,0 +1,247 @@
+        });
+
+        afterEach(() => delete CMS.URL.TEXTBOOKS);
+
+        describe("Basic", function() {
+            it("should have an empty name by default", function() {
+                return expect(this.model.get("name")).toEqual("");
+            });
+
+            it("should not show chapters by default", function() {
+                return expect(this.model.get("showChapters")).toBeFalsy();
+            });
+
+            it("should have a ChapterSet with one chapter by default", function() {
+                const chapters = this.model.get("chapters");
+                expect(chapters).toBeInstanceOf(ChapterSet);
+                expect(chapters.length).toEqual(1);
+                return expect(chapters.at(0).isEmpty()).toBeTruthy();
+            });
+
+            it("should be empty by default", function() {
+                return expect(this.model.isEmpty()).toBeTruthy();
+            });
+
+            it("should have a URL root", function() {
+                const urlRoot = _.result(this.model, 'urlRoot');
+                return expect(urlRoot).toBeTruthy();
+            });
+
+            it("should be able to reset itself", function() {
+                this.model.set("name", "foobar");
+                this.model.reset();
+                return expect(this.model.get("name")).toEqual("");
+            });
+
+            it("should not be dirty by default", function() {
+                return expect(this.model.isDirty()).toBeFalsy();
+            });
+
+            it("should be dirty after it's been changed", function() {
+                this.model.set("name", "foobar");
+                return expect(this.model.isDirty()).toBeTruthy();
+            });
+
+            return it("should not be dirty after calling setOriginalAttributes", function() {
+                this.model.set("name", "foobar");
+                this.model.setOriginalAttributes();
+                return expect(this.model.isDirty()).toBeFalsy();
+            });
+        });
+
+        describe("Input/Output", function() {
+            var deepAttributes = function(obj) {
+                if (obj instanceof Backbone.Model) {
+                    return deepAttributes(obj.attributes);
+                } else if (obj instanceof Backbone.Collection) {
+                    return obj.map(deepAttributes);
+                } else if (_.isArray(obj)) {
+                    return _.map(obj, deepAttributes);
+                } else if (_.isObject(obj)) {
+                    const attributes = {};
+                    for (let prop of Object.keys(obj || {})) {
+                        const val = obj[prop];
+                        attributes[prop] = deepAttributes(val);
+                    }
+                    return attributes;
+                } else {
+                    return obj;
+                }
+            };
+
+            return it("should match server model to client model", function() {
+                const serverModelSpec = {
```

Comment:
I just noticed that the conversion is using ES2015 syntax (``const``, in this case). I'm a little nervous that if you didn't have to do anything to switch to Babel that these files won't work as is. There were issues in the past where Karma just ignored files that it found to have syntax errors, so maybe that's why your build is passing.

I went to check the coverage report to see if this file is being tested, and it looks like it is, but there's something odd about it because it can't find the source code:

https://build.testeng.edx.org/job/edx-platform-js-pr/39440/cobertura/coffee_spec_models/cms_static_coffee_spec_models_textbook_spec_js/

It may be that there's nothing wrong but I'd like to be sure.

Note also that ESLint will reject these files once they are processed because the default linter is ES5. We'll need to have an .eslintrc.json in the root directory that says that all the files should use our ES2015 rules.